deployments/helm-chart/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: nginx-ingress
-version: 0.14.0
+version: 0.14.1
```

Comment:
It is not necessary to update our chart.yaml file. We update this with releases.

Reply:
I reverted the original version